frontend/src/pages/Alerts/LogAlert/LogAlertPage.tsx

```diff
@@ -112,7 +112,7 @@ export const LogAlertPage = () => {
 		skip: !alert_id,
 	})
 
-	const form = useFormState<LogMonitorForm>({
+	const formStore = useFormStore<LogMonitorForm>({
```

Comment:
This was happening because of unnecessary `value` assignment trying to assign `value` to `formState.values.*`. This worked with the old form states, but doesn't appear to work with the new component stores.